### PR TITLE
feat: Enable new identity attribute `verified_email`

### PR DESCRIPTION
**Pull request by aterga**
# Motivation

Verified emails help app developers by informing then whether particular emails have actually been verified.

# Changes

# Tests

Added "with verified_email attribute" e2e test for the happy scenario.

Edge cases specific to verified emails were already tested in Rust.

### CODE REVIEW

File: src/frontend/tests/e2e-playwright/authorize/openid.spec.ts [hunk 1]
```diff
@@ -143,4 +143,42 @@ test.describe("Authorize with direct OpenID", () => {
...
+
+  test.describe("with verified_email attribute", () => {
+    const email = "john.doe@example.com";
+
+    test.use({
+      openIdConfig: {
+        defaultPort: DEFAULT_OPENID_PORT,
+        createUsers: [
+          {
+            claims: { email, email_verified: "true" },
```
[the remark this answers is not in the document]
Comment (sea-snake) [reply src/frontend/tests/e2e-playwright/authorize/openid.spec.ts:155]: Good catch!

File: src/frontend/src/routes/(new-styling)/(resuming-channel)/resume-openid-authorize/+page.svelte [hunk 2]
```diff
@@ -143,8 +145,12 @@
       });
       directOpenIdFunnel.trigger(DirectOpenIdEvents.CallbackFromOpenId);
       const authFlowResult = await authFlow.continueWithOpenId(config, jwt);
+      const { name, email } = decodeJWT(jwt);
       if (authFlowResult.type === "signUp") {
-        await authFlow.completeOpenIdRegistration(authFlowResult.name!);
+        await authFlow.completeOpenIdRegistration(
+          // Prefer name, then email prefix, then fallback to a generic name
+          name ?? email?.split("@")[0] ?? $t`${config.name} user`,
```
Comment (aterga) [inline src/frontend/src/routes/(new-styling)/(resuming-channel)/resume-openid-authorize/+page.svelte:151]: Why do we need this?

File: src/frontend/src/routes/(new-styling)/(resuming-channel)/resume-openid-authorize/+page.svelte [hunk 3]
```diff
@@ -111,6 +112,7 @@
 
   onMount(async () => {
     const searchParams = new URLSearchParams(window.location.hash.slice(1));
+    console.log("Search params:", window.location.hash);
```
Comment (aterga) [inline src/frontend/src/routes/(new-styling)/(resuming-channel)/resume-openid-authorize/+page.svelte]:
```suggestion
```

File: src/frontend/src/routes/(new-styling)/(resuming-channel)/resume-openid-authorize/+page.svelte (hunk 2 above)
Comment (sea-snake) [reply src/frontend/src/routes/(new-styling)/(resuming-channel)/resume-openid-authorize/+page.svelte:151]: A JWT doesn't always have a name, particularly in the newly added test in this PR it only has an email.

Since the direct OpenID flow is non-interactive  sign-in/up, this change adds a fallback to either part of the email or a generic name, compared to previously passing undefined to a canister method that would throw an error.

In practice this isn't applicable to the OpenID providers we support in this flow, but it is during the e2e and might be in the future.
